services/ircd/inspircd/default.nix

```diff
@@ -0,0 +1,74 @@
+{config, lib, pkgs, ...}:
+with lib;
+let
+  tld = config.redbrick.tld;
+  configPath = "/etc/inspircd/inspircd.conf";
+  pkg = import ../../../packages/inspircd { inherit pkgs; };
+  attrToConfig = import ../../../common/xml.nix { inherit lib; };
+  inspircdConf = import ./conf.nix { inherit config lib; };
+  configFile = pkgs.writeText "inspircd.conf" (attrToConfig inspircdConf);
+in {
+  environment.etc."inspircd/inspircd.conf" = { source = configFile; };
```

Comment:
Is this used by something else in the system? Otherwise you can make `configPath = configFile` and remove this.

Reply:
i dont believe so